base/sparse/spqr.jl

```diff
@@ -21,133 +21,293 @@ const ORDERING_BESTAMD = Int32(9) # try COLAMD and AMD; pick best#
 # tried.  If there is a high fill-in with AMD then try METIS(A'A) and take
```

Comment:
We've never been building suitesparse with metis support - older versions of metis had a strange license.